vignettes/bambu.Rmd

```diff
@@ -232,29 +273,139 @@ and the gene expression estimates that include any newly discovered genes and
 transcripts to text files.   
 ```{r}
 save.dir <- tempdir()
-writeBambuOutput(se, path = save.dir, prefix = "NanoporeRNASeq_")
+writeBambuOutput(se.multiSample, path = save.dir, prefix = "NanoporeRNASeq_")
```

Comment:
Error in `:=`(TXNAME, "Incomaptible") : 
Check that is.data.table(DT) == TRUE. Otherwise, := and `:=`(...) are defined for use in j, once only and in particular ways. See help(":=").

Reply:
same here, when you replace se with se.multiSample, does it still give you error?